study.md

```diff
@@ -40,14 +44,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+The 'bang' symbol indicates that the object(in this case a string) will have its original state changed. In this case, 'my_string', the original object, will be capitalized.
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+the '@'symbol before a given variable will create an instance of class.
```

Comment:
This creates an instance variable.  To instantiate an object from a class, the syntax is: `class_instance = ClassName.new`.